src/platform_impl/windows/ime.rs

```diff
@@ -124,16 +124,24 @@ impl ImeContext {
             left: x,
             top: y,
             right: x + width,
-            bottom: y - height,
+            bottom: y + height,
         };
         let candidate_form = CANDIDATEFORM {
             dwIndex: 0,
             dwStyle: CFS_EXCLUDE,
             ptCurrentPos: POINT { x, y },
             rcArea: rc_area,
         };
+        let composition_form = COMPOSITIONFORM {
+            dwStyle: CFS_EXCLUDE,
```

Comment:
There's no such option for the form.

Reply:
yeah according to the official document should use CFS_POINT here. i'll take a fix